README.md

```diff
@@ -13,31 +13,21 @@ This module creates a number of SNS topics and Slack Notification modules to pro
 
 #### Required
 
-* `slack_webhook_key` - The key for your Slack webhook. This is the part of the webhook url after '/services/'.  Consider this a 'secret'.  As such, it is STRONGLY advised not to commit this into the repository, but instead use a ParameterStore data resource to retrieve the data.  Similarly, do not commit the terraform state files, but store them in an encrypted bucket.
-* `slack_channel_default` - Default channel where messages should be sent
+* `default_slack_webhook_url` - The URL for your Slack webhook. This includes `https://`.  Consider this a 'secret'.  As such, it is STRONGLY advised not to commit this into the repository, but instead use a ParameterStore data resource to retrieve the data.  Similarly, do not commit the terraform state files, but store them in an encrypted bucket.
+* `default_slack_channel` - Default channel where messages should be sent. Required if any of `default_slack_webhook_url` or `*_overrides.slack_webhook_url` are provided.
```

Comment:
These aren't technically required since there is a default value. Also where is the documentation for teams variables?

Reply:
> These aren't technically required since there is a default value.

True, but the lambda function will shit the bed when executed if not present.  Not sure if we can create a derived requirement for this... Ideally the code would fail to deploy if one is provided but the other isn't present.

> Also where is the documentation for teams variables?

<img src="https://user-images.githubusercontent.com/236700/108869036-d3a86380-75c4-11eb-9d80-228add5893aa.png" width="100" height="100" />